sdk/tables/azure-data-tables/inc/azure/data/tables/models.hpp

```diff
@@ -54,41 +53,45 @@ namespace Azure { namespace Data { namespace Tables {
      * @brief Include this parameter to specify that the tables' metadata be returned as part of
      * the response body.
      */
-    enum class ListTablesIncludeFlags
+    enum class QueryTablesIncludeFlags
     {
       None = 0,
       Metadata = 1,
     };
-    inline ListTablesIncludeFlags operator|(ListTablesIncludeFlags lhs, ListTablesIncludeFlags rhs)
+    inline QueryTablesIncludeFlags operator|(
```

Comment:
Do you want to consider adding all the logical operators for completeness?

Also, if we want to enable logical operators on bitfield enumerations, this might be a situation to relax the rules and use a #define to define these operators (there's a Windows macro that does this which I've found helpful in the past).

If it's a one-off it shouldn't be a huge issue.

Reply:
what is this macro , i could use it,